gulpfile.js

```diff
@@ -95,8 +97,8 @@ gulp.task('cssmin', function () {
 gulp.task('sass', function () {
 	return gulp.src(defaultAssets.client.sass)
 		.pipe(plugins.sass())
-		.pipe(plugins.rename(function (path) {
-			path.dirname = path.dirname.replace('/scss', '/css');
+		.pipe(plugins.rename(function (file) {
+			file.dirname = file.dirname.replace(path.sep + 'scss', path.sep + 'css');
```

Comment:
Sorry, I meant into Gulp. Also, how is it `path.sep` when we are passing in `file` now?